lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2065,6 +2068,51 @@ LogicalResult StmtEmitter::visitSV(CoverOp op) {
   return success();
 }
 
+LogicalResult StmtEmitter::visitSV(AssertConcurrentOp op) {
+  SmallPtrSet<Operation *, 8> ops;
```

Comment:
I'd recommend factoring this code better, e.g. add so you could implement this like:

```
LogicalResult StmtEmitter::visitSV(AssertConcurrentOp op) {
  return emitVerifOperation("assert", op.label(), op.event(), op.property());
}
```

or something like that.  Is this possible?

Reply:
Good thought. That would clean this up. I looked into a template while refactoring this, but it's more straightforward to just use a function. I'll clean this up for the immediate assertion variants on `main`, too.